src/java.base/share/man/java.1

```diff
@@ -279,6 +279,12 @@ the compilation.
 This sets both the source version accepted by compiler and the system
 API that may be used by the code in the source file.
 .IP \[bu] 2
+If \f[V]--enable-preview\f[R] is specified, the \f[V]--source N\f[R]
+arguments can be omitted.
+The Java Runtime of version \f[V]N\f[R] implies an implicit
```

Comment:
I think that should be `A Java runtime version ...`

Reply:
Also `implies an implicit` is doubling up. I suggest the following complete sentence:
```
If the Java runtime version is N, then `--release N` is implied when compiling source files.
```

Reply:
Done via https://github.com/openjdk/jdk/pull/17731/commits/a82e4fe3d05a411635bbce0a69e17218184e7ab6